### DISCUSSION TITLE
How to contribute translations?

### DISCUSSION THREAD

**Discussion by Albert1st1** [Heroic-Games-Launcher/HeroicGamesLauncher#274] [General]
I wonder how one should do when contributing to translations.

[Swedish.zip](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/6206102/Swedish.zip)

**Comment by BiRo96** [comment]
Check out this Wiki link for detailed description of how you can add your translation to the project :) 
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Translations

**Reply by Albert1st1** [reply]
I don't use git and i never pushed anything with git i did however add a zip file with the files translated to swedish.
I did look at this commit to see which files to translate https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/230/commits

**Comment by flavioislima** [comment]
Swedish! Yeah!
I live in Sweden but don't know swedish yet! Thanks for that, @Albert1st1 

I'll add it to the project :D

**Reply by Albert1st1** [reply]
I updated the translation.
[Swedish.zip](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/6216461/Swedish.zip)

**Comment by skowr** [comment]
Hey.
I just noticed some stupid typos in Polish translation.
Should I submit the whole pull request to update few letters?

**Reply by Albert1st1** [reply]
@flavioislima has been working on a weblate for translation so future updates should be made there.
I think it's best to wait until that work is done.
I searched for it and found 
https://hosted.weblate.org/projects/heroic-games-launcher/

**Comment by adityaruplaha** [comment]
Now its possible to contribute translations to Weblate without dealing the JSON files directly.
https://hosted.weblate.org/projects/heroic-games-launcher/

@Albert1st1 you are welcome to upload your translations there.

EDIT: Didn't notice they were already added, my bad.

**Comment by ArturWroblewski** [comment]
I added a full translation into Polish. I would like to update it with additional sentences that will appear in the final 1.8 version. How do I know if no new sentences have been added to the translation? To make it before the final version?
When will the official version 1.8 be released?
I am asking to have time to have a look at the translation page before the release. Is there a release schedule?

**Reply by flavioislima** [reply]
> I added a full translation into Polish. I would like to update it with additional sentences that will appear in the final 1.8 version. How do I know if no new sentences have been added to the translation? To make it before the final version?
> When will the official version 1.8 be released?
> I am asking to have time to have a look at the translation page before the release. Is there a release schedule?

Take a look on our weblate page. The link is on the github page.
Also, 1.8 is ongoing, we will release another release candidate before reaching a stable build, so 1.8 stable should be release by the end of July or August depending on if we finish everything planned on the road map.
You can check the road map on the github as well. 🙂

**Reply by skovorodal10n** [reply]
Hi there. 
I've pushed Ukrainian localization to 100%. Can expect it on board next update for QA?

**Reply by flavioislima** [reply]
Yes, for sure. I will add it ASAP.

**Reply by skovorodal10n** [reply]
Hello my friend. Congrats with new cool update. That were pretty entertaining days here in Ukraine so I had no time to check it on release date. Still makes me very sad and a bit angry coz no Ukrainian in settings. Can hope on next one? 

Adding any stylish Cyrillic font would also be great coz now app uses something boring like arial judging by Bulgarian.

Best wishes.

**Reply by flavioislima** [reply]
Hi @skovorodal10n . I can see that the translation are 100% I will add it on next release. :)

**Comment by torstenchr** [comment]
Hi.
I would love to contribute to the project with a Danish translation - however I seem to be too stupid to create a new language for the project :(
I have an account on weblate (linked to my github account), but it's not possible for me to create a new language and danish is not yet an option.
